helm/capa-karpenter-taint-remover/values.yaml

```diff
@@ -17,10 +17,7 @@ unwantedTaints:
 # The tolerations need to at least contain all taints defined in the `unwantedTaints` list.
 # WARNING: this is not validated by the chart, so make sure to keep them in sync.
 tolerations:
-  - key: node.cluster.x-k8s.io/uninitialized
-    operator: Exists
-  - key: node.kubernetes.io/not-ready
-    operator: Exists
+  - operator: Exists
```

Comment:
I was thinking that it would make more sense to remove this value and inline it in the daemonset definition because it needs to allow all taints anyway, and it would be one less thing to configure. Why expose it as a value if we always need to allow all taints anyway?

Reply:
yeah, sounds good to do it that way